src/drawing/backend.rs

```diff
@@ -191,15 +192,37 @@ pub trait DrawingBackend: Sized {
             return Ok(());
         }
 
-        match font.draw(text, (pos.0, pos.1), |x, y, v| {
-            self.draw_pixel((x as i32, y as i32), &color.mix(f64::from(v)))
+        let layout = font.layout_box(text).map_err(DrawingErrorKind::FontError)?;
+        let ((min_x, min_y), (max_x, max_y)) = layout;
+        let width = (max_x - min_x) as i32;
+        let height = (max_y - min_y) as i32;
+        let dx = match style.pos.h_pos {
+            HPos::Left => 0,
+            HPos::Right => -width,
+            HPos::Center => -width / 2,
+        };
+        let dy = match style.pos.v_pos {
+            VPos::Top => height,
+            VPos::Center => height / 2,
```

Comment:
I believe you have changed the `Font::draw` to use the baseline as coordinate, this is perfect. But if this is the case, we may just have a very small offset when we use the middle anchor point. I suspect this is related to the small offset in all the Y-axis label
![image](https://user-images.githubusercontent.com/3023614/69504551-0b67bd80-0ee1-11ea-89b9-58e45852004d.png)